go14/plan.sh

```diff
@@ -17,7 +17,7 @@ do_prepare() {
   build_line "Setting GOOS=$GOOS"
   export GOARCH=amd64
   build_line "Setting GOARCH=$GOARCH"
-  export CGO_ENABLED=1
+  export CGO_ENABLED=0
```

Comment:
This is indeed a better fix than the one I did recently (and which is merged already).
Maybe it could be nice to remove the `env CGO_ENABLED=0` on line 72?